subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/kotlindsl/KotlinDslVsGroovyDslPerformanceTest.groovy

```diff
@@ -35,8 +35,8 @@ class KotlinDslVsGroovyDslPerformanceTest extends AbstractCrossBuildPerformanceT
 
         given:
         runner.testGroup = 'Kotlin DSL vs Groovy DSL'
-        def groovyDslBuildName = 'Groovy DSL build'
-        def kotlinDslBuildName = 'Kotlin DSL build'
+        def groovyDslBuildName = 'Groovy_DSL_build'
+        def kotlinDslBuildName = 'Kotlin_DSL_build'
```

Comment:
What was wrong with the spaces here?

Reply:
There're issues when these spaces are passed as system properties, something like `GRADLE_OPTS=-DgradleUserHome=Groovy DSL Build`

so I simply removed the space.